pkg/ovssubnet/controller/kube/bin/openshift-sdn-kube-subnet-setup.sh

```diff
@@ -87,6 +87,9 @@ function setup_required() {
     if ! docker_network_config check; then
         return 0
     fi
+    if ovs-ofctl -O OpenFlow13 dump-flows br0 | grep -q NXM_NX_TUN_IPV4; then
+        return 0
```

Comment:
This will give the wrong result if br0 doesn't exist. The multitenant check doesn't have that problem because of how it uses a "!". Something like:

    if ! ovs-ofctl -O OpenFlow13 dump-flows br0 | grep -q 'table=0.*arp'; then
        return 0
    fi

should work.

(OK, fine, the script *already* does the wrong thing if lbr0 exists but br0 doesn't, but we might as well fix that while we're here...)